generators/spring-controller/templates/src/main/java/package/web/rest/_Controller.java

```diff
@@ -0,0 +1,35 @@
+<%#
+ Copyright 2013-2017 the original author or authors from the JHipster project.
+
+ This file is part of the JHipster project, see http://www.jhipster.tech/
+ for more information.
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+-%>
+package <%=packageName%>.web.rest;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+/**
+ * <%= controllerClass %> controller
+ */
+@RestController
+@RequestMapping("/api/<%= apiPrefix %>")
+public class <%= controllerClass %>Controller {
+
+    private final Logger log = LoggerFactory.getLogger(<%= controllerClass %>Controller.class);
+
```

Comment:
Maybe we should also add a simple "GET" method? Like a "helloworld()" method that returns a String?

Reply:
How about a similar process like the field adding in entity sub-generator? And if no method added, then add a hello-world method

Reply:
That would be much better otherwise the class we generate wouldnt have much value over the one you could create in an IDE

Reply:
yes +1 - but keep the question simple